script.php

```diff
@@ -507,6 +507,15 @@ function postflight($type, $parent)
         Log::add(Text::_('COM_JOOMGALLERY_ERROR_CREATE_DEFAULT_CONFIG'), 8, 'joomgallery');
       }
     }
+
+    // remove old JoomGallery sql files
+    foreach($this->detectSQLfiles() as $file)
+    {
+      if(File::exists($file))
```

Comment:
File::exists is deprecated and is removed in Joomla 6. Please use fule_exists() instead.

https://www.php.net/manual/en/function.file-exists.php

Reply:
@Elfangor93 Thanks for the hint. I will change that.
In order to test this and other PRs with Joomla 6, an 'upmerge' from master to the 4.2.0 branch would be helpful.
This is because the fixes for Joomla 6 are currently not included in the 4.2.0 branch.